### PR TITLE
Parallelize pytest

### PR DESCRIPTION
**Pull request by jbusecke**
Towards #223

### CODE REVIEW

**Comment by jbusecke** [comment]
This literally cuts the execution time of the tests in half. 
Before:
image

After:
image


I dont see any downside here for now. @rabernat what do you think?

**Review by dcherian — APPROVED** [review]

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -81,4 +81,4 @@ jobs:
     - name: Run Tests
       shell: bash -l {0}
       run: |
-        pytest
\ No newline at end of file
+        pytest
```
Comment (andersy005) [inline .github/workflows/ci.yaml:84]:
```suggestion
        pytest -n auto
```

Comment (jbusecke) [reply .github/workflows/ci.yaml]: Ah nice catch. Thx

**Comment by jbusecke** [comment]
Waiting for @rabernat to merge. But to me this looks good now.

**Review by dcherian — APPROVED** [review]

**Pull request merged**